src/ros_filter.cpp

```diff
@@ -1957,6 +1974,42 @@ namespace RobotLocalization
   }
 
   template<typename T>
+  bool RosFilter<T>::enableFilterSrvCallback(std_srvs::Empty::Request& request,
```

Comment:
I have to imagine that this is warning you when you build, no? I would think that you can get rid of the `request` parameter name, since it's unused (same goes for the declaration).